packages/bezier-react/src/components/Avatars/AvatarGroup/AvatarGroup.tsx

```diff
@@ -164,13 +176,18 @@ forwardedRef: React.Ref<HTMLDivElement>,
   ])
 
   return (
-    <StyledAvatarGroup
-      className={className}
-      data-testid={AVATAR_GROUP_TEST_ID}
+    <Styled.AvatarGroup
+      role="group"
       ref={forwardedRef}
-      spacing={spacing}
+      className={className}
+      style={{
+        ...style,
+        '--bezier-avatar-group-spacing': `${spacing}px`,
+        '--bezier-avatar-group-size': `${size}px`,
+      } as React.CSSProperties}
```

Comment:
```suggestion
			interface AvatarGroupCSSProperties extends React.CSSProperties {
				'--bezier-avatar-group-spacing': number
				'--bezier-avatar-group-size': number
			}

      style={{
        ...style,
        '--bezier-avatar-group-spacing': `${spacing}px`,
        '--bezier-avatar-group-size': `${size}px`,
      } as AvatarGroupCSSProperties}
```

Type assertion을 최대한 피하고 타입을 더 좁히면 좋을 것 같아서 찾아보니 Type assertion을 빼는 방법은 없는 것 같네요 😓 위에 올린 suggestion도 어차피 `React.CSSProperties`가 `Record<string, any>` 같은 타입일 것 같아서 extend해서 사용하는 게 별 의미는 없을 것 같지만, 적어도 사용하는 CSS variable을 명시할 수 있으면 좋겠다는 생각입니다.

References:

- https://github.com/frenic/csstype#what-should-i-do-when-i-get-type-errors

Reply:
이 부분은 우선 이렇게 처리하고, 좋은 방법을 더 고민해볼게요.